src/containers/tud/constants/SummarizedDataVariables.js

```diff
@@ -0,0 +1,49 @@
+// flow
+
+import { OrderedSet } from 'immutable';
+
+// These are the column headers in the csv summary data
+// TODO: verify that this is the expected order
```

Comment:
This seems like one of those TODOs that needs to be done before this PR goes live. @anzioka

Reply:
This file is no longer needed. column headers are generated from `ol.variable` values in the entity set data